packages/demo/src/app/app.component.html

```diff
@@ -1,135 +1,441 @@
 <div class="layout-container header-big">
-  <sp-intranet-header siteTitle="Swiss Post Design System" languages="en" logoUrl="/">
+  <ng-template #versionDropdown ngbDropdown>
+      <label class="visually-hidden" for="versionDropdownButton" id="versionDropdown">Version Selection for older documentation versions</label>
+      <button class="btn btn-link dropdown-button" id="versionDropdownButton" ngbDropdownToggle>
+        v{{ version }}
+      </button>
+
+      <div ngbDropdownMenu aria-labelledby="versionDropdown">
+        <a
+          *ngFor="let v of versions"
+          [href]="v.version === version ? 'https://design-system.post.ch' : v.url"
+          [class.active]="version === v.version"
+          ngbDropdownItem
+        >
+          <span class="bold">{{ version === v.version ? v.title + ' (latest)' : v.title }}</span>
+          <p class="small mt-2">
+            Angular {{v.dependencies['@angular/core'].split('.')[0]}}
```

Comment:
It's good to show the versions of the dependencies! 

However the number of versions being destined to grow, we would benefit from a more condensed display. We could for example show icons: 
![image](https://user-images.githubusercontent.com/33580481/224297282-49c8080c-e47a-4c46-b639-edec6dc69269.png)

Reply:
Done.